src/ci/github-actions/ci.yml

```diff
@@ -305,10 +298,12 @@ defaults:
     # shell is PowerShell.)
     shell: bash
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
Will this handle unrolled perf builds and regular try builds the right way? Those are both force pushed to the try / try-perf branches in rapid succession, without waiting for the previous commit to finish.

Reply:
I think that they should, because each push will have a different SHA. I will try to simulate it on my bors test repo.

Reply:
Hm, but how does this work on PRs then? There's multiple shas too. I think including a comment about this is a good idea.

Reply:
On a PR, `github.event.pull_request.number` should be non-null, therefore it will use the PR number as the concurrency key. I tried it on this PR before, and it has worked for a PR re-push. Adding a comment is a good idea, I did that.